test/verify/check-sosreport

```diff
@@ -33,7 +33,11 @@ class TestSOS(testlib.MachineCase):
         super().setUp()
         m = self.machine
 
-        if m.image.startswith("debian") or m.image.startswith("ubuntu"):
+        # Newer Debian and Ubuntu systems have sosreport in /var/tmp now
+        # https://github.com/sosreport/sos/pull/4090
+        if (m.image.startswith("debian") and not m.image.startswith("debian-testing")) or (
+            m.image.startswith("ubuntu") and not m.image.startswith("ubuntu-25.10")
```

Comment:
There will not be a 25.10; it's not an LTS, so it will be tracked by `ubuntu-stable`. Also, no dot in the name. So for now, all our ubuntu images still have /tmp.